### PR TITLE
backgrounds and gradients final

### PR DESCRIPTION
**Pull request by aaronsekisambu**
## Description
- Styling the page using gradient and background images

## How to Manually Test
- Clone the repo and fetch all branches from the origin
- Check to `background-and-gradients`
- Run the index.html file

## Screenshots
Screen Shot 2019-10-06 at 02 58 38

### CODE REVIEW

**Comment by ebukaume** [comment]
Hi @aaronsekisambu :wave:

Nice job so far; however, there are some crucial things we need to get right about this project.

- [ ] The project has a descriptive README.md. You can refer to [this repo](https://github.com/othneildrew/Best-README-Template) as a guide.
- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `.DS_Store`)

**NavBar**
- [ ] It should be made of nine different buttons with the Apple logo on the leftmost section and a search field on the rightmost section.

**Bottom**
- [ ] It contains four elements which are links (to other articles or `#`)

**Footer**
- [ ] It includes links like copyright, terms of use, and privacy policy

**Kindly submit another code review request after fixing these issues.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
[Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

**Comment by ebukaume** [comment]
Hi @aaronsekisambu 

Good job implementing some of the changes. However, we need all those other issues fixed too before we can have green on this project. :man_shrugging: 

**Your navbar is still missing the Apple Logo and it is also distorted.** 

aaron-issue-top


Also, your page looks quite different from what we have [here](http://archive.fo/UW4oR). I trust you can do better :wink: 

**Kindly submit another code review request once you have fixed those.**

Happy Coding :v: 
**Ebuka Umeokonkwo**
[Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

**Comment by aaronsekisambu** [comment]
> Hi @aaronsekisambu
> 
> Good job implementing some of the changes. However, we need all those other issues fixed too before we can have green on this project. 🤷‍♂
> 
> **Your navbar is still missing the Apple Logo and it is also distorted.**
> 
> aaron-issue-top
> 
> Also, your page looks quite different from what we have [here](http://archive.fo/UW4oR). I trust you can do better 😉
> 
> **Kindly submit another code review request once you have fixed those.**
> 
> Happy Coding ✌️
> **Ebuka Umeokonkwo**
> [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Twitter](https://twitter.com/ebukaume)

@ebukaume Thanks for the feedback once again but I Think you should pull all the latest changes from my develop branch. I already implemented those changes.


Screen-Shot-2019-10-08-at-01 09 12

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
+  <title>Background and Gradients</title>
+</head>
+
+<body>
+  <nav class="gradient-header">
+    <ul class="global-nav">
+      <li class="nav-items">
+        <a class="logo-a" href=""><span><img  class="apple-logo" src="../assets/apple_logo_PNG19673.png" alt=""></span></a>
```
Comment (Redvanisation) [inline index.html]: The logo will show up if you change the `src` attribute to `src="./assets/apple_logo_PNG19673.png"`.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @aaronsekisambu  🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The apple logo on the navbar is not showing up, I see you have it in the markup but it is not showing up on my screen, see my comment below on how to fix that.

2. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference

3. It is better is you wrap the main content of the website in a `<main>` semantic HTML tag.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Redvanisation — APPROVED** [review]
**Good work so far you have met most of the requirements and I am approving your project.**

### However and for future projects, please make sure you commit more often and to use descriptive commit messages of the changes you made on each commit.

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Comment by aaronsekisambu** [comment]
> **Good work so far you have met most of the requirements and I am approving your project.**
> 
> ### However and for future projects, please make sure you commit more often and to use descriptive commit messages of the changes you made on each commit.
> ### You may merge to master and submit a project completion form. 🚀
> ### Good luck! 🍀

@Redvanisation Having a lot of commits causes issues when rebasing in case you have conflict. That's the more reason I squash all my commits into one commit. Thanks a lot for the feedback

**Pull request merged**